browser/ui/views/frame/brave_browser_view.h

```diff
@@ -76,9 +79,19 @@ class BraveBrowserView : public BrowserView {
 #if BUILDFLAG(IS_WIN)
   bool GetSupportsTitle() const override;
 #endif
+  bool ShouldShowWindowTitle() const override;
 
   views::View* sidebar_host_view() { return sidebar_host_view_; }
 
+  const VerticalTabStripWidgetDelegateView*
+  vertical_tab_strip_widget_delegate_view() const {
+    return vertical_tab_strip_widget_delegate_view_.get();
+  }
+  VerticalTabStripWidgetDelegateView*
+  vertical_tab_strip_widget_delegate_view() {
+    return vertical_tab_strip_widget_delegate_view_.get();
+  }
+
```

Comment:
How about refer `vertical_tab_strip_widget_delegate_view_` directly from test code instead of adding public methods?

Reply:
Good. That could limit unnecessary exporsure.